### PR TITLE
Eye_detection

### PR DESCRIPTION
**Pull request by cor1234**

### CODE REVIEW

File: Eye_detection [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#Eye Detection live video
+import numpy as np
+import cv2
+
+cap = cv2.VideoCapture(0)
+#face_cascade = cv2.CascadeClassifier(cv2.data.haarcascades + 'haarcascade_frontalface_default.xml')
```
Comment (AmitSingh12345678) [inline Eye_detection:6]: Uncomment this line.

File: Eye_detection [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+eye_cascade = cv2.CascadeClassifier(cv2.data.haarcascades + 'haarcascade_eye.xml')
+
+while True:
+    ret, frame = cap.read()  
+
+    gray = cv2.cvtColor(frame, cv2.COLOR_BGR2GRAY)  
+    #faces = face_cascade.detectMultiScale(gray, 1.3, 5)
```
Comment (AmitSingh12345678) [inline Eye_detection:13]: Uncomment this line.

File: Eye_detection [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        cv2.rectangle(frame, (x, y), (x + w, y + h), (255, 0, 0), 5)
+        roi_gray = gray[y:y + w, x:x + w]  
+        roi_color = frame[y:y + h, x:x + w]  
+        eyes = eye_cascade.detectMultiScale(roi_gray, 1.3, 5)  
+        for (ex, ey, ew, eh) in eyes:
+            radius = eh//2
+            eye_x = int(ex+0.5*ew)
+            eye_y = int(ey+0.5*ey)
+            cv2.circle(roi_color, (eye_x, eye_y), radius, (0, 0, 255), 5)
+           # center_coordinates = ex + ew // 2, ey + eh // 2
```
Comment (AmitSingh12345678) [inline Eye_detection:24]: Remove unnecessary commented code.

**Review by AmitSingh12345678 — CHANGES_REQUESTED** [review]
This code still gives errors. Follow the steps mentioned below.

**Comment by cor1234** [comment]
import numpy as np
import cv2

cap = cv2.VideoCapture(0)
face_cascade = cv2.CascadeClassifier(cv2.data.haarcascades + 'haarcascade_frontalface_default.xml')
eye_cascade = cv2.CascadeClassifier(cv2.data.haarcascades + 'haarcascade_eye.xml')

while True:
    ret, frame = cap.read()  

    gray = cv2.cvtColor(frame, cv2.COLOR_BGR2GRAY)  
    faces = face_cascade.detectMultiScale(gray, 1.3, 5)  
    for (x, y, w, h) in faces:
        cv2.rectangle(frame, (x, y), (x + w, y + h), (255, 0, 0), 5)
        roi_gray = gray[y:y + w, x:x + w]  
        roi_color = frame[y:y + h, x:x + w]  
        eyes = eye_cascade.detectMultiScale(roi_gray, 1.3, 5)  
        for (ex, ey, ew, eh) in eyes:
            radius = eh//2
            eye_x = int(ex+0.5*ew)
            eye_y = int(ey+0.5*ey)
            cv2.circle(roi_color, (eye_x, eye_y), radius, (0, 0, 255), 5)
           # center_coordinates = ex + ew // 2, ey + eh // 2
           # radius = eh // 2
           # cv2.circle(roi_color, (ex, ey), radius, (0, 0, 255), 5)
         #cv2.rectangle(roi_color, (ex, ey), (ex + ew, ey + eh), (0, 255, 0), 5)

    cv2.imshow('frame', frame)

    if cv2.waitKey(1) == ord('q'):
        break

cap.release()
cv2.destroyAllWindows()

**Comment by cor1234** [comment]
import numpy as np
import cv2

cap = cv2.VideoCapture(0)
face_cascade = cv2.CascadeClassifier(cv2.data.haarcascades + 'haarcascade_frontalface_default.xml')
eye_cascade = cv2.CascadeClassifier(cv2.data.haarcascades + 'haarcascade_eye.xml')

while True:
    ret, frame = cap.read()  

    gray = cv2.cvtColor(frame, cv2.COLOR_BGR2GRAY)  
    faces = face_cascade.detectMultiScale(gray, 1.3, 5)  
    for (x, y, w, h) in faces:
        cv2.rectangle(frame, (x, y), (x + w, y + h), (255, 0, 0), 5)
        roi_gray = gray[y:y + w, x:x + w]  
        roi_color = frame[y:y + h, x:x + w]  
        eyes = eye_cascade.detectMultiScale(roi_gray, 1.3, 5)  
        for (ex, ey, ew, eh) in eyes:
            radius = eh//2
            eye_x = int(ex+0.5*ew)
            eye_y = int(ey+0.5*ey)
            cv2.circle(roi_color, (eye_x, eye_y), radius, (0, 0, 255), 5)
          

    cv2.imshow('frame', frame)

    if cv2.waitKey(1) == ord('q'):
        break

cap.release()
cv2.destroyAllWindows()

**Comment by cor1234** [comment]
Now,check I remove  all comment line

**Comment by Sanchit-sk** [comment]
Rohan, there is no need to comment the code.
Could you please push the commits after making the above changes?

**Pull request closed without merging**